### DISCUSSION TITLE
Osu!Lazer 30 fps

### DISCUSSION THREAD

**Discussion by Kenshelent** [ppy/osu#23731] [Q&A] [answered]
Osu!Lazer 30 fps on my IPhone 14 pro max, why? 
![image](https://github.com/ppy/osu/assets/38864337/046b71c0-80f8-47af-9b34-90708a9652a8)

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort, especially on mobile devices. Please be patient and wait for future optimisations, or upgrade your device.

The next release will improve things slightly, but we have more work to do to make it as smooth as we hope for.
